---
title: "Curriculum Vitae"
author: "Peter Hickey"
date: "`r BiocStyle::doc_date()`"
output:
  pdf_document: default
  html_document: null
editor_options:
  chunk_output_type: console
---

# Contact

- Email: peter.hickey@gmail.com
- GitHub: [PeteHaitch](https://github.com/petehaitch)
- Web: [www.peterhickey.org](www.peterhickey.org)
- Phone: +1-443-515-7937
- Address: Department of Biostatistics, Johns Hopkins Bloomberg School of Public Health, 615 N. Wolfe St, E3527, Baltimore, MD 21202, USA
- Australian Citizen

# Education and research experience

- *2016-present*: **Postdoctoral Fellow** Department of Biostatistics, Johns Hopkins Bloomberg School of Public Health, Baltimore, USA.
    - Advisor: [Assistant Professor Kasper Hansen](http://www.hansenlab.org/)
- *2011-2015*: **PhD (Statistics)** Department of Mathematics and Statistics, The University of Melbourne, Australia.
    - Thesis: The statistical analysis of high-throughput assays for studying DNA methylation 
    - Advisors: [Professor Terry Speed](https://www.wehi.edu.au/people/terry-speed) and Professor Peter Hall
- *2009-2015*: **Research Assistant** Bioinformatics Division, Walter and Eliza Hall Institute, Melbourne, Australia.
    - Advisor: [Professor Melanie Bahlo](https://www.wehi.edu.au/people/melanie-bahlo)
- *2008-2009*: **Undergraduate Research Opportunities Program scholar** Bioinformatics Division, Walter and Eliza Hall Institute, Melbourne, Australia.
    - Advisor: [Professor Melanie Bahlo](https://www.wehi.edu.au/people/melanie-bahlo)
- *2006-2009*: **Bachelor of Science (Mathematics and Statistics) with First Class Honours** The University of Melbourne, Australia.
    - Thesis: X chromosome association testing in genome-wide association studies
    - Advisors: [Professor Melanie Bahlo](https://www.wehi.edu.au/people/melanie-bahlo) and [Professor Richard Huggins](http://www.ms.unimelb.edu.au/Personnel/profile.php?PC_id=289)

<!---
# Research and professional interests

**TODO**: Does this belong on a CV?

- Applied statistics
- Using statistics to solve problems
- Developing and contributing to open source software to make it easier to analyse data
- Helping people make better use of available computional tools
--->

<!---
# Expertise
--->

<!---
## Additional training
--->

# Publications

\*Indicates equal contributions

```{r bib, echo = FALSE, message = FALSE, results = "hide"}
library(RefManageR)
BibOptions(max.names = Inf)

bib <- ReadBib("../bib/Peter_Hickey.bib")

myPrint <- function(bib, no.print = NULL) {
  if (length(bib) && length(no.print)) {
    for (i in seq_along(no.print)) {
      bib <- do.call(`$<-`, list(x = bib, 
                                 name = tolower(no.print[i]), value = NULL))
    }
  }
  y <- format(bib, 
              style = "markdown", 
              .bibstyle = "numeric",
              citation.bibtex.max = Inf,
              .sorting = "ydnt", 
              .sort = TRUE)
  # NOTE: A hack to add joint first authorship to a particular paper
  # TODO: Figure out a general way of doing this
  y <- gsub("P.[[:space:]]+F.[[:space:]]Hickey",
            "**P. F. Hickey**",
            y)
  i_joint <- grep("Rizzardi", y)
  y[i_joint] <- gsub("L.[[:space:]]F.[[:space:]]Rizzardi",
                     "L. F. Rizzardi\\\\*",
                     y[i_joint])
  y[i_joint] <- gsub("P.[[:space:]]F.[[:space:]]Hickey",
                     "P. F. Hickey\\\\*",
                     y[i_joint])
  # NOTE: A hack to make my name appear in bold 
  writeLines(paste(y, collapse = "\n\n"))
  invisible(list(y, bib))
}

val <- myPrint(bib)
bib <- val[[2]]
bibtypes <- vector("character", length(bib))
for (i in seq_along(bib)) {
  bibtypes[i] <- attr(unclass(bib[[i]])[[1]], "bibtype")
}
```

## Journal articles

```{r journal_articles, results = "asis", echo = FALSE}
myPrint(bib[bibtypes == "Article"], no.print = "language")
```

## Preprints

```{r preprints, results = "asis", echo = FALSE}
myPrint(bib[bibtypes == "Unpublished"], no.print = "language")
```

## Theses

```{r theses, results = "asis", echo = FALSE}
myPrint(bib[grepl("Thesis", bibtypes)], no.print = "language")
```

## Technical reports, conference papers and published abstracts

```{r published_abstracts, results = "asis", echo = FALSE}
myPrint(bib[bibtypes == "InProceedings"], no.print = "language")
```

## Commentaries and meeting reports

```{r commentaries, results = "asis", echo = FALSE}
myPrint(bib[bibtypes == "Misc"], no.print = "language")
```

<!---
## Working papers and work in progress
-->

<!---
## Blog
# TODO: Add examplar posts?
-->

# Computer skills and software development

Very proficient with statistical computing, particularly R, and strategies for 
managing and analysing large (multi-gigabyte to terabtye size) datasets, 
especially genomics data. Familiar with C, C++, and various databases — both as 
standalone tools and as integrations with and extensions of R — as well as 
Python, Unix shell tools, and cluster job schedulers.

Developer and contributor to several R packages through the Bioconductor 
project. Also developed and published Python software. Additional projects are 
available from [https://github.com/PeteHaitch](https://github.com/PeteHaitch).

```{r pkg_stats, echo = FALSE}
years <- seq(2014, 2017)
names(years) <- years
pkgs <- c("GenomicTuples", "bsseq", "minfi")
names(pkgs) <- pkgs
n_downloads <- sapply(pkgs, function(pkg) {
  sapply(years, function(year) {
    url <- paste0("http://bioconductor.org/packages/stats/bioc/", pkg, "/", 
                  pkg, "_", year, "_stats.tab")
    read.table(url, header = TRUE)[13, "Nb_of_downloads"]
  })
})
rownames(n_downloads) <- years
n_distinct_ips <- sapply(pkgs, function(pkg) {
  sapply(years, function(year) {
    url <- paste0("http://bioconductor.org/packages/stats/bioc/", pkg, "/", 
                  pkg, "_", year, "_stats.tab")
    read.table(url, header = TRUE)[13, "Nb_of_distinct_IPs"]
  })
})
rownames(n_distinct_ips) <- years
```

## R packages

Download statistics are reported for years since I have been a substantial 
author/contributor to the package (data from [http://bioconductor.org/packages/stats/](http://bioconductor.org/packages/stats/)).

### Author

- [GenomicTuples](http://bioconductor.org/packages/GenomicTuples/): Representation and Manipulation of Genomic Tuples
    - Number of downloads (2014-2017): `r n_downloads[, "GenomicTuples"]`
- [bsseq](http://bioconductor.org/packages/release/bioc/html/bsseq.html): Analyse, Manage and Store Bisulfite Sequencing Data
    - Number of downloads (2014-2017): `r n_downloads[, "bsseq"]`

### Contributor

- [minfi](http://bioconductor.org/packages/release/bioc/html/minfi.html): Analysing Illumina Infinium DNA Methylation Arrays
    - Number of downloads (2017): `r n_downloads["2017", "minfi"]`

## Python packages

### Author

- [methtuple](https://pypi.python.org/pypi/methtuple/): A caller for DNA methylation events that co-occur on the same DNA fragment from high-throughput bisulfite sequencing data, such as whole-genome bisulfite-sequencing

# Funding

<!---
**TODO** Anthing else I can list? See what other people list for ideas
--->

## Grants 

- *2015*: Bioconductor Travel Grant
    - To attend the Bioconductor meeting in Seattle, USA
- *2013*: EMBL Australia Travel Grant
    - To attend the EMBL PhD Symposium in Heidelberg, Germany

## Scholarships

- *2013*: Victorian Life Sciences Computation Initiative PhD Top-Up Scholarship.
- *2012*: Statistical Society of Australia (Victoria Branch) scholarship to attend the Young Statisticians Conference (2013)
- *2011-2015*: Australian Postgraduate Award
- *2009*: Maurice Belz scholarship, The University of Melbourne
    - A competitive scholarship awarded to complete Honours degree in statistics, stochastic processes or operations research
- *2009*: Alan W Harris Honours scholarship, The Walter and Eliza Hall Institute of Medical Research

## Awards

- *2015*: Edith Moffat Travel Award
    - To interview for international for postdoctoral positions and attend the European Bioconductor meeting
- *2013*: Prize for best lightning talk, Australian Epigenetics Conference 2013
- *2013*: Prize for third best oral presentation, Young Statisticians Conference 2013
- *2010*: Best presentation (Statistics), 2010 Victorian Mathematics and Students’ Conference

# Presentations

<!---
# TODO
- Include links to all slides/posters
- Construct bibtex file for easy autogeneration?
---> 

## Talks

### Conferences

- *2017*: **Neuronal brain region-specific DNA methylation and chromatin accessibility are associated with neuropsychiatric disease heritability.**. Contributed talk, GTEx Project Community Meeting, Rockville, USA (28/06)
- *2017*: **Developing statistical methods for large epigenomic studies in the human brain**. Contributed talk, ENAR 2017 Spring Meeting, Washington D.C., USA (13/03)
- *2016*: **New features in `bsseq` for analysing large whole genome bisulfite-sequencing datasets**. Lightning talk, BioC 2016, San Francisco, USA (24/06)
- *2016*: **The `GenomicTuples` package**. Lightning talk, BioC 2015 Developers’ Day, Seattle, USA (20/7)
- *2015*: [**Genomic tuples and DNA methylation patterns**](http://peterhickey.org/GenomicTuples_BioC_2015.html). Contributed talk, European Bioconductor Developers’ Meeting, Heidelberg, Germany (12/01)
- *2014*: [**Making sense of DNA methylation data**](https://www.youtube.com/watch?v=WVC03AH2oqs). PhD completion seminar, Melbourne, Australia (15/09)
- *2014*: [**Simulating whole-genome DNA methylation data**](http://peterhickey.org/slides/ASC2014_Peter_Hickey.pdf)). Contributed talk, Australian Statistical Conference/International Mathematical Statistics Annual Meeting, Sydney, Australia (10/07)
- *2013*: **Exploiting local dependencies in genome-wide studies of DNA methylation**. Contributed talk, Young Statisticians Conference, Melbourne, Australia (07/02)
- *2012*: **Spatial dependence of CpG-methylation from whole genome bisulfite sequencing**. Contributed talk, Epigenomics of Common Diseases Meeting, Baltimore, USA (15/10)
- *2012*: **Spatial dependence of DNA methylation**. Contributed talk, Australian Statistical Conference, Adelaide, Australia (15/07)
- *2010*: **Bioinformatics - Applied statistics in modern molecular biology**. Contributed talk (with Davis McCarthy), 2010 Victorian Mathematics and Statistics Students’ Conference, Melbourne, Australia (02/07)

### Other

- *2013*: **Bioinformatics for bisulfite sequencing**. Invited talk, La Trobe University Sequencing Users Group, Melbourne, Australia (28/08)
- *2010*: **X chromosome association testing in genome wide association studies**. Invited talk, Statistical Society of Australia - Victorian branch meeting, Melbourne, Australia (24/08)

## Posters 

- *2014*: [**Simulating whole-genome bisulfite-sequencing data**](https://figshare.com/articles/Simulating_whole_genome_bisulfite_sequencing_data/834976). Poster, Lorne Genome, Lorne, Australia (17/01)
- *2013*: [**Simulating whole-genome bisulfite-sequencing data**](https://figshare.com/articles/Simulating_whole_genome_bisulfite_sequencing_data/834976). Poster, Epigenetics 2013, Shoal Bay, Australia (03/12)
- *2011*: **Analysis of mouse exome sequencing: filtering institute specific single nucleotide variants (SNVs)**. Poster, GeneMappers - 8th Australian Human Gene Mapping Conference, Hobart, Australia (04/04)
- *2010*: **X chromosome association testing in genome wide association studies**. Poster, The International Genetic Epidemiology Society Conference, Boston, USA (12/10)
- *2010*: **X chromosome association testing in genome wide association studies**. Poster, The Australasian Microarray and Associated Technologies Association Conference, Hobart, Australia (16/09)
- *2009*: **Homozygosity by state analysis in highly inbred pedigrees**. Poster, GeneMappers - 7th Australian Human Gene Mapping Conference, Sydney, Australia (15/06)

# Teaching experience

## Workshops

- *2016*: [**Analysing DNA methylation data with Bioconductor**](https://htmlpreview.github.io/?https://github.com/hansenlab/tutorial.450k/blob/master/inst/doc/methylation450k.html). BioC 2016, Palo Alto, USA (26/06)

## Tutoring

- *2006-2014*: Private tutoring 
    - One-on-one and small-group tutoring for students studying Masters-level biostatistics, third-year university level statistics and first-year university level engineering mathematics. Tutoring senior high school students in elementary, intermediate and advanced mathematics subjects and physics.
- *2011*: Classroom teaching assistant 
    - Assisting in mathematics classes for final year high school students

# Professional activities and service

## Reviewing

Referee for *Nature Methods*, *PLoS Genetics*, *Genome Biology*, *Bioinformatics*, *PLoS Computational Biology*, *Genetic Epidemiology*, *Heredity*, and *F1000*.

## Professional society memberships

- Member, Statistical Society of Australia
- Member, International Biometric Society (ENAR)

# References

## Professor Terry Speed

Bioinformatics Division

The Walter and Eliza Hall Institute of Medical Research 1G Royal Parade, Parkville, Victoria 3052, AUSTRALIA 

Tel: +61 (0) 3 9345 2555

Fax: +61 (0) 3 9347 0852

Email: terry@wehi.edu.au

## Assistant Professor Kasper Hansen

Department of Biostatistics

Johns Hopkins Bloomberg School of Public Health 615 North Wolfe Street, Baltimore, MD 21205-2179, USA

Tel: +1-410-955-3067

Fax: +1-410-955-0958

Email: khansen@jhsph.edu

## Professor Melanie Bahlo

Population Health and Immunity Division

The Walter and Eliza Hall Institute of Medical Research 1G Royal Parade, Parkville, Victoria 3052, AUSTRALIA 

Tel: +61 (0) 3 9345 2555

Fax: +61 (0) 3 9347 0852

Email: bahlo@wehi.edu.au
