+++
title = "X Chromosome Association Testing in Genome-Wide Association Studies"
date = 2009-11-05

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["**Peter F Hickey**"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
# 7 = Thesis
publication_types = ["7"]

# Publication name and optional abbreviated version.
publication = "Honours Thesis, Department of Mathematics and Statistics, The University of Melbourne"
publication_short = "Honours Thesis"

# Abstract and optional shortened version.
abstract = "Genome wide association studies (GWAS) have revealed fascinating insights into the genetics of complex diseases. These studies provide many statistical challenges but one problem that has received surprisingly little attention is the testing of associations between phenotype and genotype on the X chromosome.\n\nIn this thesis we show that there are methods that perform significantly better than those in current wide-spread use for the analysis of X chromo- some GWAS data. In particular we establish that the methods proposed by Clayton (2008) are amongst the most powerful for X chromosome analy- sis. We quantify these gains via a simulation study under a variety of genetic models and experimental designs, to compare eight existing analytical meth- ods.\n\nUsing the knowledge gained from this simulation study we apply the most powerful method to the X chromosome data from a genome wide association study of multiple sclerosis (Australia and New Zealand Multiple Sclerosis Genetics Consortium (ANZgene), 2009). Our analysis identifies 11 genetic markers that warrant further study, an improvement upon the published analysis of this data."
abstract_short = ""

# Featured image thumbnail (optional)
image_preview = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter the filename (excluding '.md') of your project file in `content/project/`.
#   E.g. `projects = ["deep-learning"]` references `content/project/deep-learning.md`.
projects = []

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = ["statistical genetics", "undergrad"]

# Links (optional).
url_pdf = "publications/2009/Honours_Thesis.pdf"
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Does this page contain LaTeX math? (true/false)
math = false

# Does this page require source code highlighting? (true/false)
highlight = true

# Featured image
# Place your image in the `static/img/` folder and reference its filename below, e.g. `image = "example.jpg"`.
[header]
image = ""
caption = ""

+++
