---
title: "Research Statement"
author: "Peter Hickey"
date: "`r BiocStyle::doc_date()`"
output:
  pdf_document: default
  html_document: null
editor_options:
  chunk_output_type: console
---

## Research overview

My aims as an applied statistician are to provide simple, robust solutions to 
important problems and to use my creativity as a data analyst to identify 
unusual features of the data to suggest new hypotheses. I have a strong 
interest in developing statistical software and also in bringing data science 
skills and ways of thinking to researchers in underserved fields. 

Most of my research is in the interdisciplinary fields of genomics and 
bioinformatics. There, I collaborate closely with biologists and clinicians to 
understand the genetic basis of conditions such as autism spectrum disorder [1] 
and to dissect the molecular mechanisms that influence these traits, such as 
identifying differences in gene expression and chromatin state between neurons 
in different regions of the human brain [2].

I see my applied research as having two main components. Firstly, the concrete 
work to assist researchers in answering their specific questions. This includes 
helping design experiments, establishing plans for recording and managing the 
data, and using appropriate statistical methods to address their questions. The 
second component is more creative, using exploratory data analysis to identify 
unusual features in the data and, from this, to suggest hypotheses for further 
study. These two components often intersect, particularly when 
analysing data from the latest technologies for which careful and detailed 
exploratory work is required in order understand how to distinguish technical 
and biological variation.

My methodological research arises from this collaborative research, motivated 
by the pressing biological and medical questions but also by the rapid advances 
in technology that are driving the field of genomics. These new technologies 
generate vast amounts of high-dimensional data from thousands to millions of 
samples. I am interested in finding simple, robust methods to identify the 
signal in these large, noisy datasets. In my experience, this often involves 
creative extensions or repurposing of existing methodology or the recognition 
of a domain-specific problem as a special case or related case of a more 
general statistical problem. My strong statistical training and computational 
expertise makes me ideally suited to address the challenges these experiments 
provide.

I firmly believe that methodological developments can only reach their full 
potential when implemented in well-documented software that integrates with 
other relevant tools. This belief is behind my development of open source 
software and my involvement in the broader open source software community, most 
notably as a contributor to the R/Bioconductor project. Two packages that I 
have co-authored, **GenomicTuples** [3] and **bsseq** have been downloaded 
roughly 2,000 and 10,000 times in the last 12 months, respectively. My 
methodological research often requires the development of algorithms and data 
structures that scale to the analysis of gigabyte-to-terabyte sized data. I am 
contributing to core Bioconductor infrastructure to make the analysis of such 
large data routinely feasible by developing general, extensible classes for 
disk-backed data and implementing efficient methods for common operations.

The development of strong working relationships is critical to the success of 
my interdisciplinary research. I have found it vitally important to be able 
to translate concepts between statistical terminology and that used in the 
research field. My strong communication skills and interest in the broader 
scientific context of the statistical problems are appreciated by my 
collaborators. I take pride in my efforts to understand and follow the latest 
research in my collaborators' fields, which allows me to take leadership roles 
in our research. By doing so, I can move from being a statistical collaborator 
who leads the quantitative analysis of experiments, to a scientific 
collaborator who substantially contributes to the ideas and scientific 
conclusions of the research. This is exemplified by my current research in 
neuroepigenetics, where I am co-first author on our recent publication [2].

## Graduate work

My graduate work focused on the statistical analysis of high-throughput 
assays for studying DNA methylation [4]. DNA methylation is involved in 
the regulation of gene expression, embryonic development, and is dysregulated 
in many types of cancers. My PhD research involved extensive exploratory data 
analysis of data from cutting-edge technology. In doing this exploratory data 
analysis, I identified a strong spatial dependence of DNA methylation along 
individual DNA molecules, termed co-methylation, which I found is strongly 
influenced by the spacing of nucleosomes, which the DNA wraps around. In order 
to study co-methylation, I developed the **methtuple** software 
([https://github.com/PeteHaitch/methtuple](https://github.com/PeteHaitch/methtuple); 
[4]) as no existing software could extract this data from large datasets. I 
also made contributions to the widely used **bsseq** R/Bioconductor package for 
analysing DNA methylation data. Using my expertise in analysing data from these 
high-throughput assays, I was also involved in studies of the role of DNA 
methylation and other epigenetic marks in silencing genes on the X chromosome 
[5] and investigating how DNA methylation varies between littermates of 
genetically identical mice [6].

While a PhD student, I worked part-time as a research assistant, continuing my previously full-time position. In that role I collaborated with biologists and 
clinicians to map disease-causing mutations using methods from statistical 
genetics [1; 7; 8]. I developed software pipelines to process data from 
high-throughput assays, managed databases storing complex family medical 
histories, and wrote reports to explain the statistical analyses to clinicians. 
I also wrote a research paper comparing methods for analysing data from the X 
chromosome in large, genome-wide association studies of complex disease 
[9].

## Postdoctoral work

In my postdoctoral work, I continue to have an interest in analysing data on 
DNA methylation to better understand its role in different biological systems. 
With colleagues at Johns Hopkins, I have led a project in which we performed 
the most comprehensive study to date of differences in DNA methylation between 
different regions in the human brain. We have found that almost all of these 
differences occur in neuronal cells and that these 
_differentially methylated regions_ are likely important in contributing to the 
heritability of neuropsychiatric diseases such as schizophrenia and neuroticism.

During this research, I extended the method we use for identifying these 
differentially methylated regions from pairwise comparisons to handle general 
design matrices. These improvements and others were added to the **bsseq** 
R/Bioconductor package.

As the cost of these high-throughout assays continues to fall, experiments 
generating data on hundreds, and soon to be thousands, of human samples are now 
feasible. I am a part of a large consortium, the eGTEx project, where we are 
profiling DNA methylation, in conjunction with multiple other genomic 
and molecular measurements, on a wide variety of tissues from a common set of 
human donors [10]. These new datasets are many hundreds of gigabytes in 
size, too large for current approaches that typically require the data to be 
in-memory to perform an analysis. Recognising this, I am working with members 
of the Bioconductor project to develop a core set of classes, methods, and 
other infrastructure to support the analysis of disk-backed data, such as by 
storing the data on disk in the hierarchical data format (HDF5). Importantly, 
in keeping with the spirit of the Bioconductor project, the aim is to develop 
this as core infrastructure that other developers can use or extend and also to 
minimise the cognitive burden on users of these advanced methods and 
techniques. The development of these novel data structures and algorithms is 
also important in my other main postdoctoral research topic: 
_single-cell genomics_.

Most genomics data are currently generated using _bulk_ assays that report a 
signal aggregated over many tens of thousands of cells. In contrast, 
single-cell assays measure and report these signals at the level of individual 
cells. These single-cell assays provide unprecedented opportunities to examine 
rare sub-populations of cells, such as identifying treatment-resistant 
subclones within a tumour or to identify novel cell types, as well as to 
quantify the cell-to-cell variability of molecular marks in otherwise 
apparently homogeneous populations. However, compared to data from bulk assays, 
single-cell data are notably sparser, more heterogeneous, and potentially more 
liable to technical artefacts. I am working to develop statistical models for 
single-cell data that is being generated by biologists at Johns Hopkins to 
study pancreatic cancer metastasis.

Finally, in the last 2-3 years, I have also established an independent 
collaboration with sports scientists and exercise physiologists at the 
Australian Catholic University [11]. This team researches the prevalence, 
diagnosis, and treatment of sports injuries, particularly of hamstring injuries 
in elite athletes. This work combines routine statistical consultancy tasks — 
such as designing experiments and providing guidance on choice of statistical 
methods — with teaching practical data science skills (discussed further in my 
Teaching Statement).

## Future research

In the short term, my current, ongoing research projects in genomics include: 

- Improving variance estimation for the analysis of DNA methylation data. I am developing empirical Bayes methods to share information across loci to better estimate the mean-variance relationship, which can be used to shrink variance estimates
- Identifying and removing systematic biases from ATAC-seq, a high-throughput assay for measuring chromatin accessibility. Technical biases mean that certain DNA fragments are preferentially sampled in many types of high-throughput experiments, including ATAC-seq, which can lead to both false positives and false negatives
- Scaling R/Bioconductor infrastructure to handle on-disk data. I am helping plan and implement classes and methods for working with on-disk data in conjunction with members of the Bioconductor core team and members of the Bioconductor community

In the medium term, my plans include:

- To return to my PhD research studying DNA co-methylation, but using data from new 'long read' sequencing technology. A chief limitation in my PhD research was that the DNA fragments we could sequence were very short, meaning I could only investigate very local dependence structures. New nanopore technology can produce reads 100-10000× longer, which will allow more comprehensive studies of co-methylation
- Developing statistical methods that better integrate data from multiple sources. Currently, data from multiple assays are typically analysed independently of one another and the results then combined post-hoc. There is plenty of evidence, including in our recent neuroscience research [2], that this approach will miss genomic regions that might be discovered if there was a way to perform a joint analysis. However, the joint analysis is complicated by the different measurement scales, resolutions, and precisions of these assays, not to mention the intrinsic differences in variability of different molecular marks. Importantly, the successful development of such methods will immediately pay dividends as they can be used to re-analyse existing large public datasets with multiple data types

My longer term plans are more flexible, owing to my interest in addressing 
important new problems in biology with statistics being coupled with the 
rapidly changing technology driving novel biological experiments.

I continue to by fascinated by the scientific power of molecular biology, to 
enjoy the statistical and computational challenges raised by new technologies, 
and to be motivated by the potential of discoveries in this field to improve 
human health. For these reasons, I plan to continue doing statistical research 
in genomics and to collaborate with biologists and clinicians tackling 
important scientific questions. 

But I also wish to broaden my research: I am very interested in using my data science knowledge and skills to make it easier and more enjoyable for others to 
analyse data and to contribute to research and projects that serve a social 
good. I am interested in working with students of different specialities who 
are curious to develop their data analysis skills to shed light on their 
research passion.

## References

```{r bib, echo = FALSE, message = FALSE, results = "hide"}
library(RefManageR)
BibOptions(max.names = Inf)

bib <- ReadBib("../bib/Peter_Hickey.bib")

myPrint <- function(bib, no.print = NULL) {
  if (length(bib) && length(no.print)) {
    for (i in seq_along(no.print)) {
      bib <- do.call(`$<-`, list(x = bib, 
                                 name = tolower(no.print[i]), value = NULL))
    }
  }
  y <- format(bib, 
              style = "markdown", 
              .bibstyle = "numeric",
              citation.bibtex.max = Inf,
              .sorting = "ydnt", 
              .sort = TRUE)
  # NOTE: A hack to add joint first authorship to a particular paper
  # TODO: Figure out a general way of doing this
  y <- gsub("P.[[:space:]]+F.[[:space:]]Hickey",
            "**P. F. Hickey**",
            y)
  i_joint <- grep("Rizzardi", y)
  y[i_joint] <- gsub("L.[[:space:]]F.[[:space:]]Rizzardi",
                     "L. F. Rizzardi\\\\*",
                     y[i_joint])
  y[i_joint] <- gsub("P.[[:space:]]F.[[:space:]]Hickey",
                     "P. F. Hickey\\\\*",
                     y[i_joint])
  # NOTE: A hack to make my name appear in bold 
  writeLines(paste(y, collapse = "\n\n"))
  invisible(list(y, bib))
}

val <- myPrint(bib)
bib <- val[[2]]
bibtypes <- vector("character", length(bib))
for (i in seq_along(bib)) {
  bibtypes[i] <- attr(unclass(bib[[i]])[[1]], "bibtype")
}
```


```{r manual_nonsense, echo = FALSE, eval = TRUE, results = "asis"}
manual_order <- c("Brown2017", "Rizzardi2017-hi", "Hickey2017-gt", 
                  "Hickey2015-lr", "Keniry2016-qd", "Oey2015-ab", 
                  "Riley2010-am", "Phelan2016-gy", "Hickey2011-tf", 
                  "eGTEx2017", "Hickey2017-hs")
myPrint(bib[manual_order])
```

