\documentclass[11pt,letterpaper,pdf]{article}
\usepackage[T1]{fontenc}
\usepackage{fullpage}
\usepackage[utf8]{inputenc}
%% Hyperlinks and colors
\usepackage{color,hyperref, url}
\definecolor{darkblue}{rgb}{0.0,0.0,0.75}
\hypersetup{colorlinks,breaklinks,
  linkcolor=darkblue,urlcolor=darkblue,
  anchorcolor=darkblue,citecolor=darkblue}
%% Bibliography
\usepackage[autostyle]{csquotes}
\usepackage[backend=biber,
           bibstyle=numeric,sorting=ydnt,sortcites=true,natbib=true,defernumbers=true,
           maxbibnames=99,giveninits=true,uniquename=false]{biblatex}
\renewbibmacro{in:}{}
\addbibresource{hickey.bib}
\renewcommand*{\mkbibnamegiven}[1]{%
\ifitemannotation{highlight}
{\textbf{#1}}
{#1}}
% \ifitemannotation{highlight}
%   {\textbf{#1}}
%   {#1}
\renewcommand*{\mkbibnamefamily}[1]{%
\ifitemannotation{highlight}
  {\textbf{#1}}
  {#1}%
\ifitemannotation{first}
  {\textsuperscript{*}}
  {}%
\ifitemannotation{corresponding}
  {$^\dagger$}
  {}%
}%%
\usepackage{catchfile}
\newcommand{\getenv}[2][]{%
  \CatchFileEdef{\temp}{"|kpsewhich --var-value #2"}{}%
  \if\relax\detokenize{#1}\relax\temp\else\let#1\temp\fi}
\getenv[\INTERNALUSE]{INTERNALUSE}
\usepackage{changepage}
\newenvironment{myannotate}{\vspace{-\parskip}\begin{adjustwidth}{1cm}{}}{\end{adjustwidth}}
\usepackage{ifthen}
\usepackage{eqlist}
\usepackage{enumitem}
\setlength{\parindent}{0em}
\setlength{\parskip}{1ex plus0.5ex minus0.2ex}
\setlength{\fboxsep}{2.5pt}
\newcommand{\mycon}[1]{\smallskip\begin{enumerate}[resume,label={\scriptsize \arabic*$\ $},leftmargin=\parindent]\setlength{\itemsep}{#1}\vspace*{-0.7em}}
\newcommand{\ee}{\end{enumerate}}

\newcommand{\teach}[3]{%
  \vspace*{0.3\baselineskip}
  {#1}.\newline {\it #3}. #2.}

\newcommand{\talk}[4]{%
  \item #1. (#2) {\it #3} (#4).}

\newcommand{\poster}[4]{%
\item #1. (#2) {\it #3} (#4)}

% \newcommand{\grant}[7]{%
%   \vbox{%
%   \textbf{#5}\\
%   \vspace*{-1.8\baselineskip}
%   \begin{tabbing}
%     \= \hspace*{4cm} \= \hspace*{8cm} \= \kill
%     #1 \> \> #4 \> #3\\
%   \end{tabbing}
%   \vspace*{-1.8\baselineskip}
%   Principal Investigator: #2\\
%   Major Goals: #6\\
%   #7\\
%   \vspace*{5mm}
%   }
% }

\begin{document}

\hfill \today

\begin{center}
{\bf \Large CURRICULUM VITAE}\\
\vspace*{5mm}
{\Large Peter Francis Hickey}
\end{center}

% \vspace*{5mm}

\section*{PERSONAL DATA}

\begin{tabbing}
  \= Web Page:\hspace*{2cm}
  \=\href{http://www.peterhickey.org}{http://www.peterhickey.org} \\
  \> Email: \>\href{mailto:hickey@wehi.edu.au}{hickey@wehi.edu.au} \\
  \> Mailing Address: \>Walter and Eliza Hall Institute of Medical Research\\
  \> \>1G, Royal Parade\\
  \> \>Parkville VIC, 3052\\
\end{tabbing}


\section*{EDUCATION AND TRAINING}

\subsection*{Degrees}

\begin{tabbing}
  \=2015 \hspace*{1.5cm} \=Ph.D.\ in Statistics\\
  \>\>Department of Mathematics and Statistics\\
  \>\>The University of Melbourne, Melbourne \\
  \>\>Advisors: \textbf{Terry Speed} and \textbf{Peter Hall}\\
    \>2009\>B.\ Sc.\ (First Class Honours) in Mathematics and Statistics \\
    \>\>University of Melbourne
\end{tabbing}

\subsection*{Postdoctoral Training}

\begin{tabbing}
  \= 2016--2018 \hspace*{0.5cm} \= Department of Biostatistics \\
  \>\> Johns Hopkins Bloomberg School of Public Health\\
  \>\> Advisor: {\bf Kasper D.\ Hansen}
\end{tabbing}

\section*{PROFESSIONAL EXPERIENCE}

\begin{tabbing}
  \= 2018--Present \= Senior Research Officer\\
  \>\>Advanced Technology and Biology\\
  \>\>Walter and Eliza Hall Institute of Medical Research\\
  \> 2016--2018 \> Postdoctoral Fellow\\
  \>\>Department of Biostatistics\\
  \>\> Johns Hopkins University \\
  \> 2010--2015 \> Research Assistant\\
  \>\>Bioinformatics Division\\
  \>\> Walter and Eliza Hall Institute of Medical Research
\end{tabbing}

\section*{PROFESSIONAL ACTIVITIES}

\subsection*{Professional Memberships}

\begin{tabbing}
  \= Member, Statistical Society of Australia\\
  \> Member, Australasian Genomic Technologies Association\\
\end{tabbing}

% \subsection*{Project Development}

% TODO: Could I put down BioC Asia?

% \begin{tabbing}
%   \= 2012--Present \hspace*{0.5cm} \= Member of the Bioconductor Technical Advisory Board.
% \end{tabbing}

\section*{EDITORIAL ACTIVITIES}

\subsection*{Served as \textit{referee} for}

\begin{tabbing}
  \=Bioinformatics\\
  \>Biostatistics\\
  \>F1000Research\\
  \>Genetic Epigemiology\\
  \>Genome Biology\\
  \>Heredity\\
  \>Nature Methods\\
  \>PLoS Computational Biology\\
  \>PLoS Genetics\\
\end{tabbing}

\section*{HONORS AND AWARDS}

% TODO: Go through and identify others

\begin{tabbing}
  \= 2019 \hspace*{0.5cm} \= Bioconductor Travel Award (To present at the Bioconductor meeting in New York, USA)\\
  \= 2018 \hspace*{0.5cm} \= AGTA Travel Award (To present at the AGTA meeting in Adelaide, Australia)\\
  \= 2018 \hspace*{0.5cm} \= Bioconductor Travel Award (To present at the Bioconductor meeting in Toronto, Canada)\\
  \= 2015 \hspace*{0.5cm} \= Bioconductor Travel Award (To present at the Bioconductor meeting in Seattle, USA)\\
  \= 2015 \hspace*{0.5cm} \= Edith Moffat Travel Award (To interview for international for postdoctoral positions and present the European Bioconductor meeting)\\
  \= 2013 \hspace*{0.5cm} \= Prize for best lightning talk at the Australian Epigenetics Conference\\
  \= 2013 \hspace*{0.5cm} \= Third prize for best lightning talk at the Young Statisticians Conference\\
  \> 2007 \> Third prize at the Computational and Genomic Biology student retreat \\
  \>\>poster competition \\
\end{tabbing}

\section*{PUBLICATIONS}

\nocite{*}

\defbibnote{mynote}{$^*$ indicates equal contributions\\
  $^\dagger$ indicates corresponding author(s) (if not the senior author)\\
  % \textbf{boldface} indicates a member of my lab \\
  }

\printbibliography[title=Journal Articles (peer reviewed),keyword=peerjournal,prenote=mynote]

\printbibliography[title={Journal Articles, Consortia member (peer reviewed)},keyword=peerconsortia]

\printbibliography[title=Preprints (not peer reviewed),keyword=preprint,prenote=mynote]

\printbibliography[title={Theses, Editorials},keyword=others,prenote=mynote]

\printbibliography[title={Preprints, subsequently published (not peer reviewed)},keyword=pubpreprint,prenote=mynote]

\subsection*{Citation databases}

\vspace*{0.5\baselineskip}

Google Scholar: \href{https://scholar.google.com.au/citations?user=pQhJuagAAAAJ&hl=en}{profile} (link)\\
ORCID: \href{https://orcid.org/0000-0002-8153-6258}{0000-0002-8153-6258} (link)\\
Europe PMC Citations: \href{https://europepmc.org/authors/0000-0002-8153-6258}{profile} (link)

\section*{PRACTICE ACTIVITIES}

\subsection*{Software - Bioconductor Project}

% TODO: Go through and identify others

\begin{enumerate}[labelindent=1cm,align=left]
  \item[\href{http://www.bioconductor.org/packages/bsseq}{bsseq}]
    Analyze, Manage and Store Bisulfite Sequencing Data.
  \item[\href{http://www.bioconductor.org/packages/DelayedMatrixStats}{DelayedMatrixStats}]
    Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects.
  \item[\href{http://www.bioconductor.org/packages/GenomicTuples}{GenomicTuples}]
    Representation and Manipulation of Genomic Tuples.
  \item[\href{http://www.bioconductor.org/packages/minfi}{minfi}]
    Analyze Illumina Infinium DNA Methylation Arrays.
\end{enumerate}

\subsection*{Software - Other}

% TODO: Go through and identify others

\begin{enumerate}[labelindent=1cm,align=left]
  \item[\href{https://pypi.python.org/pypi/methtuple/}{methtuple}] A caller for DNA methylation events that co-occur on the same DNA fragment from high-throughput bisulfite sequencing data, such as whole-genome bisulfite-sequencing.
\end{enumerate}

% TODO: Add references? Check advice and whether application requests them.

\ifthenelse{\equal{\INTERNALUSE}{TRUE }}{

\newpage

\begin{center}
{\bf CURRICULUM VITAE}\\
Peter Francis Hickey\\[3mm]
Part II
\end{center}

\bigskip

\subsection*{TEACHING}

\subsection*{Ph.D.\ Supervision}

\begin{tabbing}
 \=Yue You (joint w/ Matt Ritchie), Medical Biology, WEHI, 2020--present.\\
 \>Shian Su (joint w/ Matt Ritchie), Medical Biology, WEHI, 2020--present.\\
\end{tabbing}

\subsection*{Undergraduate Supervision}

\begin{tabbing}
  \=Amelia Dunstone, Undergraduate Research Opportunities Program 2019--present.\\
\end{tabbing}

\subsection*{Ph.D.\ Committee}

\begin{tabbing}
  \=Aravind Manda, Population Health and Immunity, 2020--present.\\
  \>Megan Iminitoff, Epigenetics and Development Division, 2019--present.\\
\end{tabbing}

% TODO: Add supervision of William.

\subsection*{Classroom Instruction - Invited Guest Lecturer}

% TODO: Add Hopkins lecturing
% TODO: Go through and identify others
\begin{tabbing}
  \=Introduction to Single-Cell 'Omics: University of Melbourne, 2019.
\end{tabbing}


% \subsection*{Other significant teaching - Massive Open Online Courses (MOOCs)}

\subsection*{Other significant teaching - Workshops and Short Courses}

% TODO: Go through and identify talks

% \teach{Statistical Methods for Next Generation Sequencing}{%
%   2012}{ENAR Washington, D.C.} % half day
%
% \teach{Computational Statistics for Genome Biology}{%
%   2011}{Brixen, Italy} % 5 days
%
% \teach{High throughput sequence analysis tools and approaches with Bioconductor}{%
%   2009}{FHCRC, Seattle, USA}
%
% \teach{Statistical analysis of gene expression data with R and Bioconductor}{%
%   2009}{University of Copenhagen, Denmark}
%
% \teach{R for (computational) biologists}{%
%   2008}{University of California, Berkeley, USA}
%
% \teach{III International Course on Microarray Data Analysis}{%
%   2007}{Valencia, Spain}
%
% \teach{Statistical Analysis of Microarray Expression Data with R and Bioconductor}{%
%   2007}{University of Copenhagen, Denmark}
%
% \teach{I Course on Microarray Data Analysis}{%
%   2005}{Valencia, Spain}
%
% \teach{Statistical Computing with R}{%
%   2004}{University of Copenhagen}

% \section*{RESEARCH GRANT PARTICIPATION}

% \subsection*{Ongoing}

% \subsection*{Completed}

% \section*{ACADEMIC SERVICE}

\section*{PRESENTATIONS}

% TODO: Go through and identify others

\subsection*{Upcoming}
% TODO: Remove this subsection if not using

\mycon{0.3em}
\talk{empty}{}{}{}
\ee

\subsection*{Invited Talks (Seminars and Scientific Meetings)}

% TODO: Go through and identify others

% \mycon{0.3em}

% \talk{Mean-correlation relationship biases co-expression analysis}{2019}{Dept.\ of Biostatistics, VCU}{Virginia, USA}
%
% \talk{Removing Unwanted Variation Reveals the Impact of Genetic Variation on 3D Genome Structure}{2019}{JSM}{Denver, CO}
%
% \talk{Using network analysis to illuminate neurological dysfunction}{2019}{Workshop on Statistical Challenges in Medical Data Science}{Ascona, Switzerland}
%
% \talk{Co-expression patterns define epigenetic regulators associated with neurological dysfunction}{2019}{Translational Psychiatry}{Park City, Utah}
%
% \talk{Co-expression patterns define epigenetic regulators associated with neurological dysfunction}{2018}{Statistical and Computational Challenges in High-Throughput Genomics with Application to Precision Medicine (BIRS workshop)}{Oxaca, Mexico}
%
% \talk{Analyzing bisulfite sequencing data from human brain regions}{2018}{Department of Biostatistics, University of Florida}{Florida, USA}
%
% \talk{Experiences with teaching Genomic Data Science online}{2018}{JSM}{Denver, CO}
%
% \talk{Removing unwanted variation between samples in Hi-C experiments.}{2018}{Dept.\ of Mathematical Sciences}{Copenhagen, Denmark}
%
% \talk{Co-expression networks are associated with the role of the epigenetic machinery in neurological dysfunction}{2018}{Statistics in Complex Systems}{Copenhagen, Denmark}
%
% \talk{The human epigenetic machinery is very intolerant to variation and highly co-expressed}{2017}{Bioc Europe}{Cambridge, UK}
%
% \talk{Analysis of epigenetic data with biological variation}{2017}{Mid-Atlantic Bioinformatics Conference}{ Philadelphia, USA}
%
% \talk{Brain region-specific DNA methylation and chromatin accessibility}{2017}{Statistical and Computational Challenges in Large Scale Molecular Biology at BIRS}{Banff, Canada}
%
% \talk{Preprocessing issues with epigenetic assays based on sequencing}{2016}{ICSA meeting}{Atlanta, USA}
%
% \talk{Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data}{2016}{NY Epigenomics Symposium}{NY, USA}
%
% \talk{Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data}{2016}{Planning meeting for the Gilgamesh network for aging research}{Santa Fe, USA}
%
% \talk{Lessons from GTEx}{2016}{Planning meeting for the Gilgamesh network for aging research}{Santa Fe, USA}
%
% \talk{Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data}{2015}{Statistical and Computational Challenges In Bridging Functional Genomics, Epigenomics, Molecular QTLs, and Disease Genetics workshop at BIRS}{Banff, Canada}
%
% \talk{DNA Methylation Platforms}{2015}{NIA}{Baltimore, USA}
%
% \talk{Lessons from Gene Expression}{2015}{Seminar on Computational Mass Spectrometry at Dagsthul}{Saarbrucken, Germany}
%
% \talk{Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data}{2015}{UCLA}{LA, USA}
%
% \talk{Some Methods and Results concerning DNA methylation}{2015}{JHU}{Baltimore, USA}
%
% \talk{Statistical analysis of epigenetic data}{2015}{Columbia University}{NY, USA}
%
% \talk{Statistical analysis of epigenetic data}{2014}{SAMSI}{North Carolina, USA}
%
% \talk{Statistical analysis of epigenomewide data}{2014}{WNAR}{Honolulu, Hawaii}
%
% \talk{Statistical analysis of epigenetic data}{2014}{Memorial Sloan-Kettering Cancer Center}{NY, USA}
%
% \talk{Statistical analysis of epigenomewide data}{2014}{NIA}{Baltimore, USA}
%
% \talk{Statistical analysis of epigenomewide data}{2014}{University of Pittsburgh and Carnegie-Mellon University}{Pittsburgh, USA}
%
% \talk{Analysis of whole-genome bisulfite sequencing data}{2013}{Johns Hopkins University (Baltimore, USA)}
%
% \talk{A genome-wide look at DNA methylation}{2013}{Statistical Data Integration Challenges in Computational Biology: Regulatory Networks and Personalized Medicine at BIRS}{Banff, Canada}
%
% \talk{A genome-wide look at DNA methylation}{2013}{BioC2013, Bioconductor Annual Meeting}{FHCRC, USA}
%
% \talk{The structure of epigenetic changes in cancer as revealed by whole-genome shotgun bisulfite
%   sequencing}{2012}{New York University}{New York, USA}
%
% \talk{Epigenetic changes in cancer revealed by whole-genome shotgun bisulfite sequencing}{2012}{Dana-Farber Cancer Institute}{Boston, USA}
%
% \talk{Epigenetic changes in cancer revealed by whole-genome shotgun bisulfite sequencing}{2012}{University of Michigan}{Michigan, USA}
%
% \talk{Epigenetic changes in cancer revealed by whole-genome shotgun bisulfite sequencing}{2012}{Johns Hopkins University}{Baltimore, USA}
%
% \talk{The structure of epigenetic changes in cancer as revealed by whole-genome shotgun bisulfite
%   sequencing}{2012}{Johns Hopkins University}{Baltimore, USA}
%
% \talk{Epigenetic changes in cancer revealed by whole-genome shotgun bisulfite sequencing}{2012}{University of British Columbia}{Vancouver, Canada}
%
% \talk{Analysis of whole-genome shotgun bisulfite sequencing data}{2012}{University of Pennsylvania}{Philadelphia, USA}
%
% \talk{Increased methylation variation across cancer types}{2012}{Cancer Research UK}{Cambridge, UK}
%
% \talk{Epigenetic changes in cancer revealed by whole-genome shotgun bisulfite sequencing}{2012}{European Bioinformatics Institute}{Hinxton, UK}
%
% \talk{The structure of epigenetic changes in cancer as revealed by whole-genome shotgun bisulfite sequencing}{2012}{Pacific Biosciences}{Menlo Park, USA}
%
% \talk{Analysis of shotgun bisulfite sequencing of cancer samples}{2011}{Dept.\ of Mathematical Sciences}{Copenhagen, Denmark}
%
% \talk{Analysis of shotgun bisulfite sequencing of cancer samples}{2011}{Statistical Analysis of Genomic Data}{CSHL, USA}
%
% \talk{Aspects of RNA-Seq data: computations, variance and bias}{2010}{DIMACS Workshop on Next Generation Sequencing at Rutgers}{New Jersey, USA}
%
% \talk{Biological variation in high-throughput RNA sequencing experiments}{2010}{Young Investigator Symposium at Johns Hopkins}{Baltimore, USA}
%
% \talk{The use of random priming induces global biases in Illumina transcriptome sequencing}{2010}{NIH}{Bethesda, USA}
%
% \talk{Biases and variation in RNA-Seq}{2010}{Statistical Genomics in Biomedical Research workshop at BIRS}{Banff, Canada}
%
% \talk{RNA-Seq: Sequencing the Transcriptome}{2009}{High throughput sequence analysis tools and approaches with Bioconductor}{FHCRC, USA}
%
% \talk{Biases in Illumina transcriptome sequencing caused by random hexamer priming}{2009}{Gene expression based on sequencing technologies workshop}{Copenhagen, Denmark}
%
% \talk{Biases in Illumina RNA-Seq due to random priming}{2009}{University of Chicago}{Chicago, USA}
%
% \talk{Biases in Illumina RNA-Seq}{2009}{Johns Hopkins University}{Baltimore, USA}
%
% \talk{RNA-Seq: Sequencing the transcriptome}{2008}{Using Bioconductor for ChIP-Seq experiments workshop}{FHCRC, USA}
%
% \talk{RNA-Seq: Sequencing the transcriptome}{2008}{Walter and Eliza Hall Institute of Medical Research (WEHI)}{Melbourne, Australia}
%
% \talk{Investigating RNA-Seq data}{2008}{Statistical and Computational Challenges in Next-Generation Sequencing workshop}{Berkeley, USA}
%
% \talk{Modeling splice-junction arrays}{2008}{JSM}{Colorado, USA}

% \ee

% \ee


\subsection*{Scientific Meetings (Contributions)}

% TODO: Go through and identify others

% \mycon{0.3em}

% \talk{The human epigenetic machinery is very intolerant to variation and highly co-expressed}{2017}{Epigenomics of Common Disease}{Cambridge, UK}
%
% \talk{Functional Impact of Epigenomic Variation Between Individuals}{2017}{JSM}{Baltimore, USA}
%
% \talk{Choice of reference genome can introduce massive bias in bisulfite sequencing data}{2016}{Biological Data Science}{CSHL, USA}
%
% \talk{Some lessons relevant to including external libraries in your R package}{2015}{UseR}{Aalborg, Denmark}
%
% \talk{Functional Normalization}{2014}{3rd Annual Infinium HumanMethylation450 Array Workshop}{London, UK}
%
% \talk{Statistical modeling of epigenomewide data}{2013}{Joint Statistical Meetings}{Montreal, Canada}
%
% \talk{Using minfi to identify differentially methylated regions with the 450K array}{2013}{2nd Annual Infinium HumanMethylation450 Array Workshop}{London, UK}
%
% \talk{Loss of stability of epigenetic domains across cancer types}{2011}{Copenhagenomics}{Copenhagen, Denmark}
%
% \talk{Analysis of shotgun bisulfite sequencing of cancer samples}{2011}{Statistical Challenges and Biomedical Applications of Deep Sequencing Data}{Ascona, Switzerland}

% \ee

\subsection*{Posters}

% TODO: Go through and identify others

% \mycon{0.3em}

% \poster{Co-expression patterns define epigenetic regulators associated with neurological dysfunction}{2019}{Gorden Research Conference on Epigenetics}{New Hampshire, USA}
%
% \poster{Co-expression patterns define epigenetic regulators associated with neurological dysfunction}{2019}{Network Biology}{CSHL, USA}
%
% \poster{The quantitative relationship between histone modifications and gene expression across different individuals}{2016}{Biological Data Science}{CSHL, USA}
%
% \poster{Reconstructing A/B compartments as revealed by Hi-C using long-range correlations in epigenetic data}{2016}{Biology of Genomes}{CSHL, USA}
%
% \poster{The association between histone modification abundance and gene expression across individuals}{2016}{Biology of Genomes}{CSHL, USA}
%
% \poster{Reconstructing genome structure using long-range correlations in epigenetic data}{2015}{NY Epigenomics Symposium}{NY, USA}
%
% \poster{minfi: Finding differentially methylated regions using the 450k array}{2012}{Epigenomics of Common Disease}{Baltimore, USA}
%
% \poster{The structure of DNA methylation in normal tissues}{%
%   2012}{Epigenomics of Common Disease}{Baltimore, USA}
%
% \poster{Generalized loss of stability of epigenetic domains across cancers}{%
%   2011}{Statistical Methods for Very Large Datasets}{Baltimore, USA}
%
% \poster{Generalized loss of stability of epigenetic domains across cancers}{%
%   2011}{Biology of Genomes}{CSHL, USA}
%
% \poster{Biases in Illumina transcriptome sequencing caused by random hexamer priming}{%
%   2010}{MGED, ISMB satellite meeting}{Boston, USA}
%
% \poster{Biases in Illumina transcriptome sequencing caused by random hexamer priming}{%
%   2010}{HIT-Seq, ISMB satellite meeting}{Boston, USA}

% \ee

}{}
% END IF THEN

\end{document}

% Local Variables:
% LocalWords: LocalWords Biostatistics WEHI JHU
% End:
